airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -213,7 +213,7 @@ def get_conn(self) -> BlobServiceClient:
             **extra,
         )
 
-    def _get_container_client(self, container_name: str) -> ContainerClient:
```

Comment:
Otherwise MyPy fails with new azure libraries

Reply:
Not enough

Reply:
Ok. I need to ignore the errors for now - because the whole deferrab;le/non-deferrable hierarchy logic is somewhat mixed - some methods assume it's Async client and some othe that it is sync - so the hierarchy of calls will need to be reworked to make good use of new typing